api/v1/db/oracledb/member-dao.js

```diff
@@ -168,6 +169,33 @@ const deleteMember = async (memberId) => {
   }
 };
 
+const databaseName = string => (decamelize(string).toUpperCase());
+
+const isTruthyOrZero = val => (val || val === 0);
+
+const patchMember = async (memberId, attributes) => {
+  const connection = await conn.getConnection();
+  try {
+    const joinedStringArray = _.map(attributes,
+      (value, key) => (`${isTruthyOrZero(value) ? `${databaseName(key)} = :${key}` : ''}`));
+    const joinedString = _(joinedStringArray).compact().join(', ');
+    const sqlQuery = `
+    UPDATE MEMBERS
+    SET ${joinedString}
+    WHERE MEMBER_ID = :id
+    `;
+    const filteredAttributes = _.pickBy(attributes, isTruthyOrZero);
```

Comment:
From a similar comment, you should be able to filter `attributes` first and then compute `joinedStringArray` on those filtered attributes so you don't have to repeat the `isTruthyOrZero()` logic